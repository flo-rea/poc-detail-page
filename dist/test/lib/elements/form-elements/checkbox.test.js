"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const checkbox_1 = require('../../../../src/lib/view/elements/form-elements/checkbox');
jest.mock('react-dom');
describe('Checkbox', () => {
    it('has correct default options', () => {
        const component = renderer.create(React.createElement(checkbox_1.Checkbox, {id: "item2", name: "item2", checked: true}));
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=checkbox.test.js.map