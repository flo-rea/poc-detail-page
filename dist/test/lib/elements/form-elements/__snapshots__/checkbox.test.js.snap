exports[`Checkbox has correct default options 1`] = `
<input
  checked={true}
  id="item2"
  name="item2"
  type="checkbox" />
`;
